### PR TITLE
start-stop server in rds monitoring

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Closes #7

### CODE REVIEW

**Review by Polina-Gubina — DISMISSED** [review]

File: playbooks/rds_monitoring_stop_server.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  vars_files:
+    - ./vars/csm_controller.yaml
+    - ./vars/rds_monitoring.yaml
+  tasks:
+    - name: Register rds_host
+      add_host:
+        name: rds_host
+        ansible_host: "{{ rds_monitoring.ecs_ip }}"
+        ansible_ssh_user: linux
+        ansible_ssh_private_key_file: "{{ csm_controller.controller_key }}"
```
Comment (outcatcher) [inline playbooks/rds_monitoring_stop_server.yaml]: `ansible_ssh_user` and `ansible_ssh_private_key_file` should be set in inventory

File: playbooks/rds_monitoring_start_server.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  vars:
+    scenario_name: >
+      csm_controller
+      rds_monitoring
+
+- name: Prepare variables
+  hosts: localhost
+  vars_files:
+    - ./vars/csm_controller.yaml
+    - ./vars/rds_monitoring.yaml
```
Comment (outcatcher) [inline playbooks/rds_monitoring_start_server.yaml:13]: Var files are missing

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: playbooks/rds_monitoring_start_server.yaml (hunk 2 above)
Comment (anton-sidelnikov) [reply playbooks/rds_monitoring_start_server.yaml:13]: this files creates by prepare_variables.yaml

File: playbooks/rds_monitoring_stop_server.yaml (hunk 1 above)
Comment (anton-sidelnikov) [reply playbooks/rds_monitoring_stop_server.yaml]: fixed

**Review by outcatcher — APPROVED** [review]

**Pull request merged**